src/SDKs/Azure.Base/data-plane/Azure.Base.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using Azure.Base.Attributes;
+
+[assembly: AzureSdkComponent("AzureSDK.Tests")]
```

Comment:
I think we officially want to call these "Azure SDK Client Library"

Reply:
Changed to `AzureSdkClientLibraryAttribute`